sfa_dash/templates/forms/cdf_forecast_upload_form.html

```diff
@@ -0,0 +1,48 @@
+{% extends "dash/data.html" %}
+{% set page_title = 'Upload Forecast data' %}
+{% block content %}
+{% include "sections/notifications.html" %}
+{{ metadata | safe }}
+<h3>Upload data for Forecast 'AC Power'</h3>
+<form action="{{ url_for('forms.upload_cdf_forecast_data', uuid=uuid) }}" method="post" id="cdf-forecast-upload-form" enctype="multipart/form-data">
+  <div class="form-group">
+	<div class="form-element">
+	  <label>My data is formatted in:</label><br>
+	  <input type="radio" data-toggle="collapse" data-target=".upload-example" name="file-format" value="csv" checked/>CSV
+      <input type="radio" data-toggle="collapse" data-target=".upload-example" name="file-format" value="json"/>JSON
+	</div>
+	<div class="form-element full-width">
+      <div class="upload-example csv show">
+		<p>Forecast data in CSV format should follow the formatting of the example below.</p>
+	    <pre class="example-text">
+# optional header, ignored by Solar Forecast Arbiter 
+timestamp,value
+2018-11-22T12:01:48Z,10.23 
+2018-11-22T12:07:38Z,10.67</pre>
```

Comment:
Where is this copy pasta coming from? Forecasts and observations should both be at rounded times. So 12:00:00 and 12:05:00 in this example.